Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/TabBean.java

```diff
@@ -167,11 +168,11 @@ public String setDirectTabIndexNavigate(int directTabIndex) {
      */
     public void onDirectTabChange(TabChangeEvent tEvent) {
         Tab selectedTab = tEvent.getTab();
-        if (selectedTab.getId().equalsIgnoreCase(directDomainTab)) {
+        if (selectedTab.getId().equalsIgnoreCase(DIRECTDOMAINTAB)) {
```

Comment:
Optional: use DIRECTDOMAINTAB.equalsIgnoreCase(selectedTab.getId()) to prevent NPE if second operator is null